packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton.tsx

```diff
@@ -43,15 +38,21 @@ export default function RainbowKitCustomConnectButton() {
               }
 
               if (chain.unsupported || chain.id !== configuredChain.id) {
-                return (
-                  <div className="rounded-md shadow-lg p-2">
-                    <span className="text-error mr-2">Wrong network selected - ({chain.name})</span>
-                    <span className="text-primary mr-2">Switch network to</span>
-                    <button className="btn btn-xs btn-primary btn-outline" onClick={onSwitchNetwork}>
-                      {configuredChain.name}
-                    </button>
-                  </div>
-                );
+                if (chain.id === 1) {
```

Comment:
Why are you adding this check? @KcPele

Reply:
its because when it's on mainnet it does not show the wrong network button. But it works fine on any other network. i can take it off and it will still work fine